apps/api-extractor/src/analyzer/AstImport.ts

```diff
@@ -2,6 +2,26 @@
 // See LICENSE in the project root for license information.
 
 import { AstSymbol } from './AstSymbol';
+import { InternalError } from '@microsoft/node-core-library';
+
+/**
+ * Indicates the import kind for an `AstImport`.
+ */
+export enum AstImportKind {
+  /**
+   * An import statement such as `import { X } from "y";`.
+   */
+  Normal,
```

Comment:
`Normal` is a weirdly opinionated term to use. I recommend changing this term to `Bracketed`.

Reply:
Not everyone uses the same conventions we do. They aren't "abnormal."

Reply:
`Bracketed` doesn't work because this form is also used for the "default" symbol such as `import X from "Y";`

The AST calls this a `ts.SyntaxKind.NamedImports` (versus `ts.SyntaxKind.NamespaceImport` for the `* as` import).

I'll change the enum to `AstImportKind.Named`.